src/Toolkit/Obj.php

```diff
@@ -62,14 +63,27 @@ public function __get(string $property)
     }
 
     /**
-     * Property Getter
+     * Gets one or multiple properties of the object
      *
-     * @param string $property
-     * @param mixed $fallback
+     * @param string|array $property
+     * @param mixed $fallback If multiple properties are requested:
+     *                        Associative array of fallback values per key
      * @return mixed
      */
-    public function get(string $property, $fallback = null)
+    public function get($property, $fallback = null)
     {
+        if (is_array($property)) {
+            if (!is_array($fallback)) {
```

Comment:
Ah, I now know why the CI doesn't pass. If `$fallback` is `null`, it's not an array. Let's try this:

```suggestion
            if ($fallback === null) {
                $fallback = [];
            }

            if (!is_array($fallback)) {
```